src/views/preview/formsy-project-updater.jsx

```diff
@@ -41,6 +43,7 @@ class FormsyProjectUpdater extends React.Component {
         }, (err, body, res) => {
             if (res.statusCode === 200) {
                 this.setState({value: body[this.props.field], error: false});
+                this.props.onUpdate(jsonData);
```

Comment:
Most correct would be to update with body itself, in case we modified the input for some reason. I don't think that applies in any case though, so this is good for now